### ISSUE TITLE
Works fine until empty exif :(

### ISSUE THREAD

**Issue by njbuch** [saschiwy/HeicConverter#5]
Thanks for doing this little CLI. :)

It fails for me with:
```
Traceback (most recent call last):
  File "heicConverter.py", line 102, in <module>
  File "heicConverter.py", line 88, in convert_heic_to_jpeg
  File "heicConverter.py", line 44, in convert_heic_file
KeyError: 'DateTime'
[19952] Failed to execute script 'heicConverter' due to unhandled exception!
```

**Comment by saschiwy** [comment]
Do you have by any chance an example file on which it crashes?

**Comment by saschiwy** [comment]
https://github.com/saschiwy/HeicConverter/tree/Fix-handling-of-empty-exif-issue-%235

Can you verify if this fixes your issue?

**Comment by saschiwy** [comment]
Fixed with: [Release v0.1.0](https://github.com/saschiwy/HeicConverter/releases/tag/v0.1.0)
